circle.yml

```diff
@@ -8,7 +8,8 @@ dependencies:
     - nvm install 4
     - nvm install 6
     - nvm install 8
-    - rm -rf node_modules
+    - rm -rf node_modules coverage .nyc_output coverage.lcov
```

Comment:
No need to remove these new files.

Reply:
mmm am just wondering in anycase if those folder contain some older reports from different commits/codes etc and then we should not remove those before new testbed "PRE" build step so it can fetch/upload only new & updated reports for the new expected commits/changes/codes etc? 

In anycase `report-coverage": "nyc report --reporter=lcov > coverage.lcov && codecov` this will always create and rebuild those dir/files before producing reports & upload them over codecov.

So i am trying to understand here , those dir/files with old reports & meta data should be still there even with the fresh build for new changes?

Reply:
Yes, these files will be overwritten in each build anyway, that's why it makes no sense to remove them beforehand.